docs/function/functions.md

```diff
@@ -41,6 +41,7 @@ SELECT name, SUBSTRING(name, 1, 5) AS extract FROM commits
 SELECT STUFF("GQL tutorial!", 13, 1, " is fun!")
 SELECT RIGHT("AmrDeveloper", 3) AS extract
 SELECT TRANSLATE("Amr[Dev]{eloper}", "[]{}", "()()")
+SELECT UNICODE("AmrDeveloper")
```

Comment:
Missing documentations for `UNICODE` function on String functions table above

Reply:
Sorry about that. Fixed!

Reply:
No problem, thanks